index.php

```diff
@@ -45,6 +45,21 @@
         <li><a href="https://github.com/kaincenteno/wiki">wiki Github</a>
       </ul>
 
+    <h4>FOUND A BUG IN THE GAME</h4>
+      <p>If you have found a bug in the game. We encourage you to submit it through a discussion.
```

Comment:
Formatting, suggest this is changed to:
We encourage you to submit it through a discussion. If you have found a bug in the game: